cmd/validate.go

```diff
@@ -116,9 +118,10 @@ func newCheckSumCommand() *cobra.Command {
 					s := sha256.Sum256(data)
 					if !bytes.Equal(s[:], file.Sha256) {
 						return errors.Errorf(`
-backup data checksum failed: %s may be changed
-calculated sha256 is %s,
-origin sha256 is %s`, file.Name, hex.EncodeToString(s[:]), hex.EncodeToString(file.Sha256))
+								backup data checksum failed: %s may be changed
+								calculated sha256 is %s,
+								origin sha256 is %s`,
```

Comment:
Please remove the leading `\t`s, otherwise, the log looks weird.

```
								backup data checksum failed: %s may be changed
```

Reply:
Fixed